src/pages/messaging/agent-chat-api/v3.2/index.mdx

```diff
@@ -882,12 +916,18 @@ Grants access to a new chat without overwriting the existing ones.
 
 #### Specifics
 
-|                        |                                                                                        |
-| ---------------------- | -------------------------------------------------------------------------------------- |
-| **Method URL**         | `https://api.livechatinc.com/v3.2/agent/action/grant_chat_access`                      |
-| **Required scopes**    | `chats--all:rw` `chats--access:rw` `chats--my:rw`                                      |
-| **RTM API equivalent** | [`grant_chat_access`](/messaging/agent-chat-api/v3.2/rtm-reference/#grant-chat-access) |
-| **Webhook**            | [`chat_access_granted`](/management/configuration-api/v3.2/#chat_access_granted)       |
+|                        |                                                                                              |
+| ---------------------- | -------------------------------------------------------------------------------------------- |
+| **Method URL**         | `https://api.livechatinc.com/v3.2/agent/action/grant_chat_access`                            |
+| **Required scopes**    | `chats--all:rw`<sup>1</sup> or `chats--access:rw`<sup>2</sup> or `chats--my:rw` <sup>3</sup> |
+| **RTM API equivalent** | [`grant_chat_access`](/messaging/agent-chat-api/v3.2/rtm-reference/#grant-chat-access)       |
+| **Webhook**            | [`chat_access_granted`](/management/configuration-api/v3.2/#chat_access_granted)             |
+
+1) `chats--all:rw` - to grant access to a chat taking place in any group.
+
+2) `chats--access:rw` - to grant access to a chat taking place in groups that the requester (related to the token) is a member of. (The requester groups and the current groups of the chat must overlap - at least one group must be in common.)
```

Comment:
very long description, I know that it describes in detail how it works but this is also true for e.g. `deactivate_chat` and other methods; perhaps we can shorten it somehow and use such description consistently, wdyt?